src/cli/lint/internalLints/upgrade/patches/10.1.0/stopBundlingInCDKTests.test.ts

```diff
@@ -0,0 +1,124 @@
+import memfs, { vol } from 'memfs';
+
+import type { PatchConfig } from '../..';
+import { configForPackageManager } from '../../../../../../utils/packageManager';
+
+import {
+  hasAppImportRegex,
+  tryStopBundlingInCDKTests,
+} from './stopBundlingInCDKTests';
+
+const volToJson = () => vol.toJSON(process.cwd(), undefined, true);
+
+jest.mock('fs-extra', () => memfs);
+jest.mock('fast-glob', () => ({
+  glob: (pat: any, opts: any) =>
+    jest.requireActual('fast-glob').glob(pat, { ...opts, fs: memfs }),
+}));
+
+beforeEach(() => vol.reset());
+
+describe('stopBundlingInCDKTests', () => {
+  const baseArgs = {
+    manifest: {} as PatchConfig['manifest'],
+    packageManager: configForPackageManager('yarn'),
+  };
+
+  afterEach(() => jest.resetAllMocks());
+
+  describe.each(['lint', 'format'] as const)('%s', (mode) => {
+    it('should skip if no CDK test files are found', async () => {
+      await expect(
+        tryStopBundlingInCDKTests({
+          ...baseArgs,
+          mode,
+        }),
+      ).resolves.toEqual({
+        result: 'skip',
+        reason: 'no CDK test files found',
+      });
+
+      expect(volToJson()).toEqual({});
+    });
+
+    it('should skip if no App import is found from aws-cdk-lib', async () => {
+      const input = `const myApp = new App();`;
+
+      vol.fromJSON({
+        'apps/my-worker/infra/myWorkerStack.test.ts': input,
+      });
+
+      await expect(
+        tryStopBundlingInCDKTests({
+          ...baseArgs,
+          mode,
+        }),
+      ).resolves.toEqual({
+        result: 'skip',
+        reason: 'no CDK test files need patching',
+      });
+
+      expect(volToJson()).toEqual({
+        'apps/my-worker/infra/myWorkerStack.test.ts': input,
+      });
+    });
+
+    it('should patch a detected bare App() constructor', async () => {
+      const input = `import { App } from "aws-cdk-lib";\nconst app = new App();\n\nother stuff\n\nconst secondApp = new App();`;
+
+      const inputVolume = {
+        'infra/myWorkerStack.test.ts': input,
+      };
+
+      vol.fromJSON(inputVolume);
+
+      await expect(
+        tryStopBundlingInCDKTests({
+          ...baseArgs,
+          mode,
+        }),
+      ).resolves.toEqual({
+        result: 'apply',
+      });
+
+      expect(volToJson()).toEqual(
+        mode === 'lint'
+          ? inputVolume
+          : {
+              'infra/myWorkerStack.test.ts': `import { App } from "aws-cdk-lib";\nconst app = new App({ context: { 'aws:cdk:bundling-stacks': [] } });\n\nother stuff\n\nconst secondApp = new App({ context: { 'aws:cdk:bundling-stacks': [] } });`,
+            },
+      );
+    });
+  });
+});
+
+describe('hasAppImportRegex', () => {
+  it.each([
+    ['Bare', 'import { App } from "aws-cdk-lib";'],
+    ['Other after', 'import { App, aws_sns } from "aws-cdk-lib";'],
```

Comment:
This works fine, but for documentation:

```suggestion
    ['Others after', 'import { App, aws_sns, aws_sqs } from "aws-cdk-lib";'],
```